google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -374,11 +381,16 @@ interfaces:
     - DEADLINE_EXCEEDED
   - name: one_plus_delivery
     retry_codes:
+    - ABORTED
+    - CANCELLED
+    - INTERNAL
```

Comment:
@pongad , INTERNAL is certainly a complicated status to handle, as @jba clearly points out,  internal errors must be caused by invariants which usually surface by inputs that were not expected by the system and will permanently fail; a bug in the system. However we often see *internal* errors that surface just because an unexpected failure coming from an upstream dependency or an environment issue that usually gets resolved by itself; a transient error, such errors should genuinely be reported as unavailable errors but given our quite complex infrastructure, that is not always the case. Hence, in the pubsub library we have opted to a more conservative approach of retrying even on internal errors.